core/src/com/unciv/logic/trade/TradeEvaluation.kt

```diff
@@ -148,17 +149,29 @@ class TradeEvaluation {
             TradeType.Introduction -> return introductionValue(civInfo.gameInfo.ruleset)
             TradeType.WarDeclaration -> {
                 val civToDeclareWarOn = civInfo.gameInfo.getCivilization(offer.name)
-                val threatToThem = Automation.threatAssessment(civInfo, civToDeclareWarOn)
 
                 return if (!civInfo.isAtWarWith(civToDeclareWarOn)) 0 // why should we pay you to go fight someone...?
-                else when (threatToThem) {
+                else when (Automation.threatAssessment(civInfo, civToDeclareWarOn)) {
                     ThreatLevel.VeryLow -> 0
                     ThreatLevel.Low -> 0
                     ThreatLevel.Medium -> 100
                     ThreatLevel.High -> 500
                     ThreatLevel.VeryHigh -> 1000
                 }
             }
+
+            TradeType.PeaceDeclaration -> {
+                val civToMakePeaceWith = civInfo.gameInfo.getCivilization(offer.name)
+
+                return when (Automation.threatAssessment(civInfo, civToMakePeaceWith)) {
+                    ThreatLevel.VeryLow -> 1000
```

Comment:
Why should the AI be interested in this offer...? A human wouldn't be? "Hi I'm going to make peace with X please give me money" - sounds like no